3-UsingAPIs/Week2/assignment/ex3-rollAnAce.js

```diff
@@ -17,21 +17,26 @@ import { rollDie } from '../../helpers/pokerDiceRoller.js';
  * @param {DieFace} desiredValue
  * @returns {Promise<DieFace>}
  */
-export function rollDieUntil(desiredValue) {
-  // TODO rewrite this function using async/await
-  return rollDie().then((value) => {
-    if (value !== desiredValue) {
-      return rollDieUntil(desiredValue);
+export async function rollDieUntil(desiredValue) {
+  while (true) {
```

Comment:
An infinite loop is not the best approach here. We know what the loop condition should be:
```suggestion
  let value;
  while (value !== desiredValue) {
```